utils/callData/registrationCalls.ts

```diff
@@ -141,7 +141,7 @@ function altcoinRenew(
       durationInYears * 365,
       sponsor ?? 0,
       discountId ?? 0,
-      "0x" + metadataHash,
+      metadataHash,
```

Comment:
So everywhere we use metadataHash we could type it like so

```ts
type HexString = `0x${string}`;
```

 To not make the error of forgetting the "0x" (which is very probable tbh cause we did it in the past).